bulk-model-sync-gradle-test/build.gradle.kts

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2023.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import org.modelix.model.server.Main
+
+buildscript {
+    val modelixCoreVersion: String = file("../version.txt").readText()
+    dependencies {
+        classpath("org.modelix:model-server:$modelixCoreVersion")
+        classpath("org.modelix:graph-lang-api:$modelixCoreVersion")
+    }
+}
+
+plugins {
+    alias(libs.plugins.kotlin.jvm)
+    id("org.modelix.bulk-model-sync")
+}
+
+val modelixCoreVersion: String = file("../version.txt").readText()
+
+version = modelixCoreVersion
+
+repositories {
+    mavenLocal()
+    maven { url = uri("https://repo.maven.apache.org/maven2") }
+    maven { url = uri("https://plugins.gradle.org/m2/") }
+    mavenCentral()
+    maven { url = uri("https://artifacts.itemis.cloud/repository/maven-mps/") }
+}
+
+val mps by configurations.creating
+val mpsDir = buildDir.resolve("mps").apply { mkdirs() }
+val kotlinGenDir = buildDir.resolve("metamodel/kotlin").apply { mkdirs() }
+
+dependencies {
+    mps("com.jetbrains:mps:2021.2.5")
+    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.7.2")
```

Comment:
Woudn't it make sense to put this into the version catalog, too?

Reply:
Will do that in the next PR.